webpack.dev.js

```diff
@@ -22,6 +22,13 @@ module.exports = (env) => {
         },
       ],
     },
+    performance: {
+      assetFilter: function (assetFilename) {
+        return assetFilename.endsWith('.js') || assetFilename.endsWith('.css');
+      },
+      maxEntrypointSize: 3512000,
+      maxAssetSize: 3512000,
+    },
```

Comment:
this block cleans up warnings/errors in devServer when testing production mode

Reply:
Is the max asset size 3.5 MB?